pages/common/paste.md

```diff
@@ -0,0 +1,23 @@
+# paste
+
+> Merge lines of files.
+
+- join all the lines into one, using TAB as delimiter.
+
+`paste -s {{file}}`
+
+- join all the lines into one, using the specified delimiter.
+
+`paste -s -d {{delimiter}} {{file}}`
+
+- merge two files side by side, using TAB as delimiter. Lines in the first file and the second file form the first column and the second column, respectively.
```

Comment:
Think about how to make the description shorter.

E.g.
- Merge two files side by side, each in its column, using TAB as delimiter.

Or propose your example